src/containers/org/components/PermissionsPanel.js

```diff
@@ -200,7 +246,18 @@ const PermissionsPanel = ({
         {`These are the properties that are assigned the ${_lowerCase(permissionType)} permission.`}
       </Typography>
       <Divider isVisible={false} margin={24} />
-      <PropertyTypesCard>
+      <PermissionCard>
+        <PermissionCardSegment>
+          <Typography variant="body1">All Properties</Typography>
+          <IconButton onClick={togglePermissions}>
```

Comment:
please add the `aria-label` attribute to buttons that don't contain text. The label should clearly describe the destination, purpose, function, or action for screen reader users.

Reply:
Doesn't seem resolved yet